integration-tests/src/test/groovy/com/okta/sdk/tests/it/UserTypesIT.groovy

```diff
@@ -34,7 +34,7 @@ class UserTypesIT extends ITSupport {
 
     @Test
     void createUserTypeTest() {
-        String name = "java_sdk_user_type_" + RandomStringUtils.randomAlphanumeric(15)
+        String name = "java_sdk_it_" + RandomStringUtils.randomAlphanumeric(15)
```

Comment:
`UserType` creation does not support `-` in name. Therefore, resorted to snake case here.